scripts/list_needed_libs.sh

```diff
@@ -10,7 +10,14 @@ set -euo pipefail
 # (staticx knows to pack the libraries used by the executable we're packing. it doesn't know
 # which executables are to be used by it)
 
-BINS=$(find gprofiler/resources -executable -type f)
+EXCLUDED_DIRECTORIES=('"gprofiler/resources/node/*"')
```

Comment:
Do we need the exclude for an array of dirs and not just `find gprofiler/resources -executable -type f -not -path gprofiler/resources/node/`?

Reply:
We can, but I wanted it to be easy to add another directory if someone needs to do that in future. If you want I can just add it as one argument to command

Reply:
Fair. Let's keep it